assets/src/components/customers/CustomerDetailsPrimarySection.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import {Box} from 'theme-ui';
+import {Tabs} from '../common';
+import CustomerDetailsCard from './CustomerDetailsCard';
+import CustomerDetailsConversations from './CustomerDetailsConversations';
+
+const {TabPane} = Tabs;
+
+enum TAB_KEYS {
+  Conversations = 'Conversations',
+  Notes = 'Notes',
+}
+
+type Props = {customerId: string};
+type State = {};
+
+class CustomerDetailsPrimarySection extends React.Component<Props, State> {
```

Comment:
Super open to calling this something else, but couldn't figure out a name. What do you call this???

![image](https://user-images.githubusercontent.com/1361509/115077138-8dd49100-9ecb-11eb-8f6d-3b3d626c99fc.png)